go/private/skylib/lib/collections.bzl

```diff
@@ -14,60 +14,56 @@
 
 """Skylib module containing functions that operate on collections."""
 
-
 def _after_each(separator, iterable):
-  """Inserts `separator` after each item in `iterable`.
-
-  Args:
-    separator: The value to insert after each item in `iterable`.
-    iterable: The list into which to intersperse the separator.
+    """Inserts `separator` after each item in `iterable`.
```

Comment:
I'd prefer not to make any local changes to our copy of Skylib (everything under go/private/skylib). These changes will be erased next time we update.

It looks like the latest version of Skylib has been formatted with Buildifier. I'll update that in a separate PR.